volatility/framework/plugins/windows/dlllist.py

```diff
@@ -25,13 +32,65 @@ def get_requirements(cls) -> List[interfaces.configuration.RequirementInterface]
             requirements.ListRequirement(name = 'pid',
                                          element_type = int,
                                          description = "Process IDs to include (all other processes are excluded)",
-                                         optional = True)
+                                         optional = True),
+            requirements.BooleanRequirement(name = 'dump',
+                                            description = "Extract listed processes",
+                                            default = False,
+                                            optional = True)
         ]
 
+    @classmethod
+    def dump_dll(cls,
```

Comment:
We could potentially rename this to dump_pe since it is also used by modules/modscan and those aren't technically DLLs, but its not a big deal.